nautobot_device_lifecycle_mgmt/templates/nautobot_device_lifecycle_mgmt/hardwarenotice_device_report.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load static %}
+{% load helpers %}
+
+{% block content %}
+<div class="pull-right noprint">
+<div class="btn-group">
+    <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <span class="mdi mdi-plus-thick" aria-hidden="true"></span>Execute<span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu">
+            <li><a href="{% url 'extras:job_run_by_class_path' class_path='nautobot_device_lifecycle_mgmt.jobs.lifecycle_reporting.DeviceHardwareNoticeFullReport' %}">Run Hardware Notice Reporting</a></li>
+    </ul>
+</div>
+{% if request.user.is_authenticated and table_config_form %}
+    <button type="button" class="btn btn-default" data-toggle="modal" data-target="#ObjectTable_config" title="Configure table"><i class="mdi mdi-cog"></i>Configure</button>
+{% endif %}
+{% if request.user.is_authenticated and 'export' in action_buttons %}
+    {% export_button content_type %}
+{% endif %}
+</div>
+    <h1>{% block title %}Device Hardware Notice Reports{% endblock %}</h1>
+    <div class="row">
+        <div class="col-md-9">
+            <!-- VISUAL BLOCK -->
+            {% if report_last_run is None %}
+                <h4 class="text-center alert-danger p-4 m-4">-- No validation results found, you need to run the report at least once before seeing the results! --</h4>
+            {% else %}
+            <h4 class="text-left alert-info p-4 m-4">Last full run of the report: {{ report_last_run }} - {{ report_last_run|timesince }} ago </h4>
+            {% endif %}
+            {% if bar_chart is not None %}
+                {% block graphic  %}
+                    <div id="content">
+                        <img src="data:image/png;base64,{{ bar_chart|safe }}" style="width:100%" alt="Platform Bar Chart">
+                    </div>
+                {% endblock %}
+            {% else %}
+                    <span class="text-center alert-danger p-4 m-4">-- Oops, no config validation found, visual not made! --</span>
```

Comment:
```suggestion
                    <span class="text-center alert-danger p-4 m-4">-- Can't generate visualization, no config validation found. --</span>
```

Reply:
This template was copied from other templates. I will commit this suggestion and update the other templates to match.